AU2/frontends/inquirer_cli.py

```diff
@@ -208,11 +209,23 @@ def render(html_component, dependency_context={}):
         assassins_mapping = dependency_context[dependent]
         if not assassins_mapping:
             return {html_component.identifier: [], "skip": True}
+
+        assassins_in_event = [(aId, *rest) for (aId, *rest) in html_component.assassins if aId in assassins_mapping]
+        options = list(itertools.chain.from_iterable([
+            [(aId, i, p) for (i, p) in enumerate(pseudonyms)] for (aId, pseudonyms) in assassins_in_event]))

```

Comment:
It is slightly cursed that this includes deleted pseudonyms. Imo it would be better to filter them out here, except for any that appear in `default_options`.

E.g.
```suggestion
        options = list(itertools.chain.from_iterable([
            [(aId, i, p) for (i, p) in enumerate(pseudonyms) if p or (aId, i, p) in default_options]
            for (aId, pseudonyms) in assassins_in_event]))
```
after moving the code filling out `default_options` above this.